x-pack/filebeat/module/cisco/ftd/_meta/fields.yml

```diff
@@ -0,0 +1,92 @@
+- name: ftd
+  type: group
+  description: >
+    Fields for Cisco Firepower Threat Defense Firewall.
+  fields:
+  - name: message_id
+    type: keyword
+    description: >
+      The Cisco FTD message identifier.
+
+  - name: suffix
+    type: keyword
+    example: session
+    description: >
+      Optional suffix after %FTD identifier.
+
+  - name: source_interface
+    type: keyword
+    description: >
+      Source interface for the flow or event.
+
+  - name: destination_interface
+    type: keyword
+    description: >
+      Destination interface for the flow or event.
+
+  - name: rule_name
+    type: keyword
+    description: >
+      Name of the Access Control List rule that matched this event.
+
+  - name: source_username
+    type: keyword
+    description: >
+      Name of the user that is the source for this event.
+
+  - name: destination_username
+    type: keyword
+    description: >
+      Name of the user that is the destination for this event.
+
+  - name: mapped_source_ip
```

Comment:
Good catch. I changed the wrong comment to refer to the nat fields. It is already copied to ECS, but I can't remove it because the pipeline is shared with ASA and removing those fields would be a breaking change, so for now both fields, `cisco.{asa|ftd}.mapped_xxx` and ECS nat will be set.